src/irr/asset/CGLIWriter.cpp

```diff
@@ -166,16 +165,26 @@ namespace irr
 				return width * texelBlockByteSize * height * depth;
 			};
 
+			const auto ROW_TEXEL_OR_BLOCK = 0; // TODO
+			const auto blockDimension = getBlockDimensions(imageInfo.format);
+			const auto sourceTexelOrBlockStride = getTexelOrBlockBytesize(imageInfo.format) * ROW_TEXEL_OR_BLOCK;
```

Comment:
stride is in physical units (multiples of blocks/texels, although to be fair texels are just 1x1x1 blocks) not bytes